packages/fxa-auth-server/lib/routes/utils/third-party-events.ts

```diff
@@ -266,21 +266,18 @@ export async function getGooglePublicKey(
  * Validate a JWT security token against public key.
  *
  * @param token
- * @param clientId
  * @param publicKeyPem
  * @param issuer
  * @returns {Promise}
  */
 export async function validateSecurityToken(
   token: string,
-  clientId: string,
   publicKeyPem: any,
   issuer: string
 ) {
   // Decode the token, validating its signature, audience, and issuer
   return jwt.verify(token, publicKeyPem, {
     algorithms: ['RS256'],
-    audience: [clientId],
```

Comment:
Without validating the aud, couldn't someone take one of their security tokens and hand it to us?

Reply:
Good call @chenba I've updated this PR to have it read in env values that can contain supported audiences.